src/components/DeeplinkWrapper/index.website.js

```diff
@@ -48,6 +48,10 @@ class DeeplinkWrapper extends PureComponent {
         if (!this.isMacOSWeb() || CONFIG.ENVIRONMENT === CONST.ENVIRONMENT.DEV) {
             return;
         }
+
+        // We need to clear the old short-lived auth token if it exists.
```

Comment:
Why do we need to do this?

Reply:
Please update the code comment to explain the "why" and not just "what" the code is doing. Why do we need to clear out the previous one before fetching a new one?

Reply:
I think that's a perfect explanation 👍 Here is how I would write it:

> Since there is no way to know if a previous short-lived authToken is still valid, any previous short-lived authToken must be cleared out and a new one must be fetched so that the popup window will only open when we know the short-lived authToken is valid.